### PR TITLE
Added ittnotify tasks for training of Linear Regression CPU/GPU

### PR DESCRIPTION
**Pull request by prutskov**

### CODE REVIEW

File: algorithms/kernel/linear_model/linear_model_train_normeq_finalize_impl.i [hunk 1]
```diff
@@ -66,6 +68,7 @@ Status FinalizeKernel<algorithmFPType, cpu>::compute(const NumericTable &xtxTabl
 
         if (&xtxTable != &xtxFinalTable)
         {
+            DAAL_ITTNOTIFY_SCOPED_TASK(computeFinalize.copyToOnline);
```
Comment (michael-smirnov) [inline algorithms/kernel/linear_model/linear_model_train_normeq_finalize_impl.i]: This task has the same name as the one on line 82. It`s confusing for me, do you really need to consider these parts of code as a single task? Also, the name **copyToOnline** says nothing to me, could you please name it more clear?

File: algorithms/kernel/linear_model/linear_model_train_normeq_update_impl.i [hunk 2]
```diff
@@ -87,12 +88,16 @@ Status ThreadingTask<algorithmFPType, cpu>::update(DAAL_INT startRow, DAAL_INT n
...
+    {
+        DAAL_ITTNOTIFY_SCOPED_TASK(computeUpdate.syrkX);
+        Blas<algorithmFPType, cpu>::xxsyrk(&up, &notrans, &nFeatures, &nRows, &alpha,
+                                        const_cast<algorithmFPType *>(x), &nFeatures, &alpha,
+                                        _xtx, &_nBetasIntercept);
+    }
 
     if (nFeatures < _nBetasIntercept)
     {
+        DAAL_ITTNOTIFY_SCOPED_TASK(computeUpdate.gemm1X);
```
Comment (michael-smirnov) [inline algorithms/kernel/linear_model/linear_model_train_normeq_update_impl.i]: Is this really a gemm stage?

Comment (prutskov) [reply algorithms/kernel/linear_model/linear_model_train_normeq_update_impl.i]: Yes, it is equivalent of gemm with vector of ones from GPU version of implementation

File: algorithms/kernel/linear_model/linear_model_train_normeq_finalize_impl.i (hunk 1 above)
Comment (prutskov) [reply algorithms/kernel/linear_model/linear_model_train_normeq_finalize_impl.i]: fixed

**Comment by prutskov** [comment]
make test

**Pull request merged**